src/main/kotlin/ru/driics/aitrade/application/orchestrator/UpdateCycleOrchestrator.kt

```
@@ -0,0 +1,204 @@
+
+    /**
+     * Logs a compact summary of AI decisions instead of verbose JSON.
+     */
+    private fun logAiDecisionsSummary(decisions: AiTradeDecisionMap) {
+        try {
+            log.info { "═══ AI Decisions Summary (${decisions.size} symbols) ═══" }
+
+            decisions.forEach { (symbol, envelope) ->
+                val args = envelope.args
+                val signal = args.signal.uppercase()
+                val confidence = args.confidence?.let { String.format("%.2f", it.toDouble()) } ?: "N/A"
```

detekt (1.23.8)

[warning] String.format("%.2f", it.toDouble()) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)